code/modules/language/language_holder.dm

```diff
@@ -53,7 +53,18 @@ Key procs
 	var/owner
 
 /// Initializes, and copies in the languages from the current atom if available.
-/datum/language_holder/New(_owner)
+///datum/language_holder/New(_owner) //ORIGINAL
```

Comment:
```suggestion
//	/datum/language_holder/New(_owner) //ORIGINAL
```
probable vsc shenanigans

Reply:
Remove the 3 slashes. That's autodoc format.